listenbrainz/webserver/static/js/src/user/Listens.tsx

```diff
@@ -243,9 +248,17 @@ export default class Listens extends React.Component<
     }
   };
 
-  receiveNewPlayingNow = (newPlayingNow: string): void => {
-    const playingNow = JSON.parse(newPlayingNow) as Listen;
-    playingNow.playing_now = true;
```

Comment:
We used to manually add the playing_now = true, IIRC because not all playing now listened we received had it, and we rely on that field in other components (ListenCard).
Are you 100% sure this is not needed anymore?